Build breaking due to incorrect dependency resolution

Building fails since a8b6b69 due to [chrome-aws-lambda requiring puppeteer-core 10.1.*](https://github.com/alixaxel/chrome-aws-lambda#versioning).

```node
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: eleventy-api-screenshot@1.0.0
npm ERR! Found: puppeteer-core@13.4.1
npm ERR! node_modules/puppeteer-core
npm ERR!   puppeteer-core@"^13.4.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer puppeteer-core@"^10.1.0" from chrome-aws-lambda@10.1.0
npm ERR! node_modules/chrome-aws-lambda
npm ERR!   chrome-aws-lambda@"^10.1.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/alexandersandberg/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alexandersandberg/.npm/_logs/2022-03-06T11_11_34_128Z-debug-0.log
```

## Solution
Downgrade puppeteer-core.

--- a/package.json
+++ b/package.json
@@ -21,6 +21,6 @@
   "dependencies": {
     "@netlify/functions": "^1.0.0",
     "chrome-aws-lambda": "^10.1.0",
-    "puppeteer-core": "^13.4.0"
+    "puppeteer-core": "^10.1.0"
   }
 }

